src/pudl/metadata/fields.py

```diff
@@ -397,7 +397,10 @@
     },
     "cooling_id_eia": {
         "type": "string",
-        "description": "The cooling system identification number used by EIA.",
+        "description": (
+            "The cooling system identification number reported to EIA. This value may "
+            "not be a unique identifier"
```

Comment:
I actually find this addition confusing, as it is presumably unique within a given plant? I don't think it's a necessary addition personally.

Reply:
I also don't understand what this means.  I would have guessed that cooling system ID was a unique ID within a plant in the same way that generator ID or boiler ID is unique within a plant, so it would be part of a composite primary key that includes the plant ID.  Is that not true?  Or if the emissions control equipment is primarily associated with boilers, then is the unique composite primary key `(plant_id_eia, boiler_id, cooling_id_eia)`?

I would have assumed the same thing across all of the other environmental equipment IDs below which you've included the same disclaimer for.  Is that wrong?

Reply:
Ok I agree, I will remove.